SAR_Training/English/Master/ISCE3_Sentinel1_RTC.ipynb

```diff
@@ -0,0 +1,461 @@
+{
```

Comment:
<p>In my experience, defining functions in notebooks is a bit of an antipattern. It makes sense to me that these would be defined in a separate helper file that is included in the repo -- however, this does slightly decrease the visibility of the code (which could be helped by specifically noting that it's a local function call in a comment or something). What do you think?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ASFOpenSARlab/opensarlab-notebooks/pull/205/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>